### PR TITLE
Comments 1

### PR DESCRIPTION
**Pull request by SomeYoGuyeee**
Comments added to GetAllOrgs.go, client.go and parentOrg.go

### CODE REVIEW

File: database/handlers/parentOrg.go [hunk 1]
```diff
@@ -101,7 +137,7 @@ func (client Client) GetOrganizationByID(id string) *database.ParentOrg {
 	}
 	defer rowsRs.Close()
 
-	orgs, err := parseResultSetToSlice(rowsRs)
+	orgs, err := parseResultSetToSlice(rowsRs) // this fn is defined below
```
Comment (EshaanAgg) [inline database/handlers/parentOrg.go:140]: Either use `function` or `func`.

**Review by EshaanAgg — CHANGES_REQUESTED** [review]

**Comment by EshaanAgg** [comment]
You can just add new commits to this branch itself, they will be updated in this PR automatically :)

**Comment by SomeYoGuyeee** [comment]
Ok bhaiya, try karta hu

**Comment by SomeYoGuyeee** [comment]
I've made the commits bhaiya

File: database/handlers/client.go [hunk 2]
```diff
@@ -12,7 +12,13 @@ type Client struct {
 	*sql.DB
 }
 
-func New() Client {
+
+/* The following function creates a new client object instance on which methods
+ * are called to read data from the database.
+ *
+ * returns: client (database) object
```
Comment (EshaanAgg) [inline database/handlers/client.go:19]: Capitalize `R`

Comment (SomeYoGuyeee) [reply database/handlers/client.go]: Done bhaiya

**Review by EshaanAgg — APPROVED** [review]
LGTM @SomeYoGuyeee! Good work.

**Pull request merged**